src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.builds

```diff
@@ -9,6 +9,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TargetGroup>netcore50aot</TargetGroup>
```

Comment:
agree. but we'll need to pump the library version too.

---
In reply to: [81700343](https://github.com/dotnet/corefx/pull/12339#discussion_r81700343) [](ancestors = 81700343)